tests/sles4sap/publiccloud/hana_sr_test_secondary.pm

```diff
@@ -35,22 +36,29 @@ sub run {
     my $db_action = get_var('DB_ACTION', $run_args->{hana_test_definitions}{$self->{name}});
     croak('Database action unknown or not defined.') if ($db_action !~ /^(stop|kill|crash)$/);
 
-    if (($db_action eq 'crash') && get_var('HA_SBD_START_DELAY')) {
+    if (($db_action eq 'crash') && defined(get_var('HA_SBD_START_DELAY'))) {
         # Setup sbd delay in case of crash OS to prevent cluster starting too quickly after reboot.
-        $self->setup_sbd_delay(get_var('HA_SBD_START_DELAY'));
+        $self->setup_sbd_delay();
```

Comment:
Can this  crash test works also without `HA_SBD_START_DELAY` ?

Reply:
Thank you. Very good catch. In case of Public cloud tests this is done by:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/fc55bfe85aa448966df178ad0928821e5509dae5/lib/sles4sap_publiccloud.pm#L449
You can see it here for example: https://openqaworker15.qa.suse.cz/tests/196654#step/Crash_site_a-primary/24
Delay was set to 120s, normally it is 66s I think.

However on prem tests are missing this. I will create a ticket.

Reply:
TEAM-8151

Reply:
Correction. THe On prem tests do not yet have feature to set the delay value. I will change the ticket for that. So it's not a bug but a new feature, kind of...

Reply:
> Correction. THe On prem tests do not yet have feature to set the delay value. I will change the ticket for that. So it's not a bug but a new feature, kind of...

Thanks!